submissions/team_api.py

```diff
@@ -187,6 +192,21 @@ def get_team_submission(team_submission_uuid):
     return TeamSubmissionSerializer(team_submission).data
 
 
+def get_team_submission_from_individual_submission(individual_submission_uuid):
+    """
+    Returns a single, serialized, team submission for the linked individual submission uuid.
+
+    Raises:
+        - TeamSubmissionNotFoundError when no such team submission exists.
+        - TeamSubmissionInternalError if there is some other error looking up the team submission.
+    """
+    team_submission_query = TeamSubmission.objects.filter(submissions__uuid=individual_submission_uuid).all()
+    if len([team_submission_query]) < 1:
+        raise TeamSubmissionNotFoundError
+
+    return TeamSubmissionSerializer(team_submission_query[0]).data
```

Comment:
Proposing a cleaner version using `first()`:

```suggestion
    team_submission = TeamSubmission.objects.filter(submissions__uuid=individual_submission_uuid).first()
    if not team_submission:
        raise TeamSubmissionNotFoundError

    return TeamSubmissionSerializer(team_submission).data
```

Reply:
Nice. I like it!